radio/src/translations/cz.h

```diff
@@ -965,6 +965,7 @@
 #define TR_GLOBALVAR                   "Glob. proměnná"
 #define TR_MIXSOURCE                   "Zdroje mixeru"
 #define TR_CONSTANT                    "Konstanta"
+#define TR_PREFLIGHT_POTSLIDER_CHECK   "On","Off","Auto"
```

Comment:
Zap, Vyp, Auto

Reply:
If changing the order from Off, On, Auto to On, Off, Auto would require code changes I'm not the person to do it. 
I'm not a programmer and have pretty much reached my coding limit already in this PR.

Reply:
I'll look into the order so that it is consistent in all languages, seems it isn't correct right now.